src/index.test.ts

```diff
@@ -1,65 +1,68 @@
 import { EventEmitter } from "events";
 
-import type Hermione from "hermione";
+import type Testplane from "testplane";
+import type { Config } from "testplane";
 
 import plugin from "./";
 import * as readToken from "./read-token";
 
+type BrowserConfig = Config["browsers"][string];
+
 jest.mock("./config", () => ({ parseConfig: jest.fn(opts => opts) }));
 
 jest.mock("./read-token");
 const readTokenMock = readToken as jest.Mocked<typeof readToken>;
 
-describe("hermione-oauth", () => {
-    const browser = (config: object = {}): Hermione.BrowserConfig => config as unknown as Hermione.BrowserConfig;
+describe("@testplane/oauth", () => {
+    const browser = (config: object = {}): BrowserConfig => config as unknown as BrowserConfig;
 
-    const hermioneMock = (browsers: Record<string, Hermione.BrowserConfig>): Hermione => {
-        const emitter = new EventEmitter() as unknown as Hermione;
+    const testplaneMock = (browsers: Record<string, BrowserConfig>): Testplane => {
+        const emitter = new EventEmitter() as any;
```

Comment:
Why did you decide to add `ant` here instead of the old variant?

Reply:
Because later we are trying to set some properties of emitter. And we can't do that since hermione@8: those are readonly

Reply:
Because later we are trying to set some properties of emitter. And we can't do that since hermione@8: those are readonly